src/main/default/classes/Manager.cls

```diff
@@ -1,4 +1,7 @@
-// Based on https://github.com/joeferraro/MavensMate-Templates/blob/master/ApexClass/TriggerHandler.cls
+/*
```

Comment:
Good bit of refactoring here just to clean things up a bit and to get all the `On Exception` stuff hooked up.  Mostly it was just moving everything to use the `HandlerConfig`